### PR TITLE
BrainAtlas GUI polish

### PR DESCRIPTION
**Pull request by egolol**

### CODE REVIEW

**Comment by annacanal** [comment]
> I think it would be better to have as default surface colormap: white or gray, instead of the colorful parula (very nice, but can be confusing)

Another thing, when I close the main BrainAtlas window, the plotBrainAtlas and its settings window don't close. Is this intended? plotBrainAtlas and settings open and close together.

It looks everything good in general, a part of missing a white background ^^

File: braph2genesis/src/atlas/_BrainAtlas.gen.m [hunk 1]
```diff
@@ -35,7 +35,7 @@
 'BrainSurface'
 %%%% ¡gui!
 surf = get_from_varargin('human_ICBM152.nv', 'SURF', varargin{:});
```
Comment (giovannivolpe) [inline braph2genesis/src/atlas/_BrainAtlas.gen.m:37]: why do you need this?

File: braph2genesis/src/atlas/_BrainAtlas.gen.m [hunk 2]
```diff
@@ -35,7 +35,7 @@
...
-pl = PlotPropBS('EL', ba, 'PROP', BrainAtlas.SURF);
+pl = PlotPropBS('EL', ba, 'PROP', BrainAtlas.SURF, 'bkgcolor', [1 .96 .79]);
```
Comment (giovannivolpe) [inline braph2genesis/src/atlas/_BrainAtlas.gen.m:38]: The colour should default on the standard color, not be imposed here

**Review by giovannivolpe — CHANGES_REQUESTED** [review]
Also, call them PPBrainAtlasIDict
and PPBrainSurf
It'll make it easier in the long run

**Pull request merged**